src/server/services/pickup-queue/statuses/ready-up/create-pickup.js

```diff
@@ -103,7 +104,15 @@ export default async function createPickup(props) {
         status: 'waiting',
         classes: mapValues(
           pickupQueue.classes,
-          classPlayers => classPlayers.filter(player => !players.includes(player.id)),
+          classPlayers => classPlayers.filter((player) => {
```

Comment:
Why is this change needed?

Reply:
I was getting "players.include" is not a function errors everytime a pickup was created.

Reply:
Totally forgot that players is an object with the classes inside